eras/conway/impl/src/Cardano/Ledger/Conway/UTxO.hs

```diff
@@ -1,27 +1,40 @@
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# OPTIONS_GHC -Wno-orphans #-}
 
-module Cardano.Ledger.Conway.UTxO () where
+module Cardano.Ledger.Conway.UTxO (
```

Comment:
We need to add `Cardano.Ledger.Conway.UTxO` to `exposed-modules` in the cabal file, since now we export something from it, besides orphan instances